templates/document.html.slim

```diff
@@ -49,22 +49,24 @@ html lang=(attr :lang, 'en' unless attr? :nolang)
     - if attr? :stem
       - eqnums_val = (attr 'eqnums', 'none')
       - eqnums_val = 'AMS' if eqnums_val == ''
```

Comment:
MathJax 3 is more strict about invalid values. This is what I did in the browser extension:

https://github.com/asciidoctor/asciidoctor-browser-extension/blob/436ad4a387e4ce72a2df8bde1334bc8bc8b26efa/app/js/converter.js#L20-L28

As far as I remember, the value must be `none`, `all` or `ams` (in lowercase) otherwise an error is thrown.

Reply:
Thanks, using the code from the browser extension helped me to spot another mistake.

As the configuration values are touchy I've added some error handling, I've seen in the extension a default value is used but I think in a command-line tool raising an error is more acceptable and avoid bad surprises.

And as the logic start to become quite convoluted I moved the code to a helper. As I probably spent too much writting Java and I'm  loosy typist I love using constants so I've used some to store the possible values, but I can go back to literals if you prefer.

Reply:
It's alright 👌🏻